gui/canvaspainter/v7/qt5/rootqt5.cpp

```diff
@@ -0,0 +1,275 @@
+/****************************************************************************
+**
+** Copyright (C) 2016 The Qt Company Ltd.
```

Comment:
This (c) and license looks wrong given the file's content. Can you replace it by a copy&paste of the usual ROOT file header, please?